spec/models/concerns/hyku_addons/user_behavior_spec.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+RSpec.describe HykuAddons::UserBehavior, type: :model do
+  subject(:user) { create(:user) }
```

Comment:
Is it possible to use `build_stubbed`?

Reply:
@thenapking i dont think so, as its testing DB scopes.